Moesif.Middleware/NetCore/Helpers/LoggerHelper.cs

```diff
@@ -21,36 +21,38 @@ namespace Moesif.Middleware.NetCore.Helpers
 {
     public class LoggerHelper
     {
-        public async static void LogDebugMessage(bool debug, String msg)
+        ILogger _logger;
+        public LoggerHelper(ILogger logger)
         {
-            if (debug)
-            {
-                await Console.Out.WriteLineAsync(msg);
-            }
+            _logger = logger;
+        }
+        public void LogDebugMessage(bool debug, String msg)
+        {
+            _logger.LogDebug(msg);
         }
 
-        public static string GetConfigStringValues(Dictionary<string, object> moesifOptions, String configName, string defaultValue)
+        public string GetConfigStringValues(Dictionary<string, object> moesifOptions, String configName, string defaultValue)
```

Comment:
is the removal of `static` going to cause any backwards compatibility issue.. meaning dont think this should be used directly outside of this library?

Reply:
Nope.